tools/ci_build/github/azure-pipelines/linux-flake8.yml

```diff
@@ -0,0 +1,12 @@
+jobs:
```

Comment:
@pranavsharma  @tianleiwu Please review.

Reply:
Do we need a separate job for this? Can this be accommodated as part of Windows or Linux CI right after the checkout step?

Reply:
Okay, is there a way to locally run/validate the yml script, before pushing the changes?

Reply:
A developer could always run this command 'flake8 tools/ci_build/build.py' locally, right? 
If you really want to catch this as part of CI, better to add it to an existing CI job, although it sounds a bit overkill given that this code doesn't really run in production.

Reply:
Yes, but people don't always do that. Anyways, I can remove this.

Reply:
I have removed the yml file and the build succeeds as well, can you review the changes?